src/styles.scss

```diff
@@ -1,15 +1,17 @@
 /* You can add global styles to this file, and also import other style files */
-@import '~@hmcts/ccd-case-ui-toolkit/dist/cut-styles/components/typography';
-@import 'shims';
-@import 'conditionals';
-@import 'measurements';
+// @import '~@hmcts/ccd-case-ui-toolkit/dist/cut-styles/components/typography';
+// @import 'shims';
+// @import 'conditionals';
+// @import 'measurements';
 
-$black: #323232;
+// $black: #323232;
 
-$path: "/assets/";
+// $path: "/assets/";
 
-$fa-font-path: "/node_modules/font-awesome/fonts";
-$govuk-frontend-toolkit-image-path: "/assets";
-$govuk-template-image-path: "/assets";
+// $fa-font-path: "/node_modules/font-awesome/fonts";
+// $govuk-frontend-toolkit-image-path: "/assets";
+// $govuk-template-image-path: "/assets";
 
-@import "govuk-elements";
+// @import "govuk-elements";
+
+@import './style/app.scss';
```

Comment:
I suppose commented out lines could be removed as they seemed to have been moved to app.scss